src/main/java/org/currencygoldexchangeapp/constants/LoggerFactory.java

```diff
@@ -0,0 +1,20 @@
+package org.currencygoldexchangeapp.constants;
+
+import java.util.logging.ConsoleHandler;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+public class LoggerFactory {
+    private static final Logger LOGGER = Logger.getLogger(LoggerFactory.class.getName());
```

Comment:
This will result in all loggers having the same origin class, not really a good idea

Reply:
I agree, LoggerFactory has been removed